fmn/fmn_fasshim.py

```diff
@@ -23,11 +24,34 @@
 default_url = 'https://admin.fedoraproject.org/accounts/'
 creds = config.app_conf['fas_credentials']
 
-fasclient = fedora.client.fas2.AccountSystem(
-    base_url=creds.get('base_url', default_url),
-    username=creds['username'],
-    password=creds['password'],
-)
+if config.get('fasjson'):
+    client = fasjson_client.Client(url=creds.get('base_url', default_url))
+else:
+    client = fedora.client.fas2.AccountSystem(
+        base_url=creds.get('base_url', default_url),
+        username=creds['username'],
+        password=creds['password'],
+    )
+
+def make_fasjson_cache(**config):
+    log.warn("Building the FASJSON cache into redis.")
+    if _cache.get('fas_cache_built'):
+        log.warn("FASJSON cache already built into redis.")
+        return
+    global client
+    try:
+        page_number = 0
+        next_page_exists = True
+        while next_page_exists:
+            page_number +=1
```

Comment:
Flake8 will be unhappy here too. If this cache construct is frequent, maybe we could migrate it to `fasjson_client` itself, no?

Reply:
I thought about that yeah - but I was unsure because I wasn't sure what the client's request would look like. Would it just be the current `get_users` but with pagination built in?